crates/frontend/Cargo.toml

```diff
@@ -3,6 +3,28 @@ name = "frontend"
 version = "0.14.0"
 edition = "2021"
 
+# [package.metadata.wasm-pack.profile.dev]
+# # Should `wasm-opt` be used to further optimize the wasm binary generated after
+# # the Rust compiler has finished? Using `wasm-opt` can often further decrease
+# # binary size or do clever tricks that haven't made their way into LLVM yet.
+# #
+# # Configuration is set to `false` by default for the dev profile, but it can
+# # be set to an array of strings which are explicit arguments to pass to
+# # `wasm-opt`. For example `['-Os']` would optimize for size while `['-O4']`
+# # would execute very expensive optimizations passes
+# wasm-opt = ['-Os']
+#
+# [package.metadata.wasm-pack.profile.release]
+# # Should `wasm-opt` be used to further optimize the wasm binary generated after
+# # the Rust compiler has finished? Using `wasm-opt` can often further decrease
+# # binary size or do clever tricks that haven't made their way into LLVM yet.
+# #
+# # Configuration is set to `false` by default for the dev profile, but it can
+# # be set to an array of strings which are explicit arguments to pass to
+# # `wasm-opt`. For example `['-Os']` would optimize for size while `['-O4']`
+# # would execute very expensive optimizations passes
+# wasm-opt = ['-Os']
+
```

Comment:
Why add this?

Reply:
Was trying different optimization strategies for wasm-pack, will remove this, or maybe we can keep it if we want to pin on some optimization profile